lib/chef/resource/chef_acl.rb

```diff
@@ -147,7 +147,7 @@ def current_acl(acl_path)
           unless @current_acls.key?(acl_path)
             @current_acls[acl_path] = begin
               rest.get(rest_url(acl_path))
-                                      rescue Net::HTTPServerException => e
```

Comment:
I think we have an open issue here for this you can close out now

Reply:
That already got done, i was working off an old checkout of master somehow